CleanProFinder/CleanProFinder.Server/Controllers/RequestController.cs

```diff
@@ -137,5 +137,19 @@ public async Task<IActionResult> CancelRequest(Guid id, CancellationToken cancel
             var result = await _mediator.Send(request, cancellationToken);
             return ConvertFromServiceResponse(result);
         }
+
+        /// <summary>
+        /// Change requests's status.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        [HttpPost("change-request-status/{id}")]
```

Comment:
Please add some description in documentation comments so that it more clear what it do. Also try to find more suitable name for endpoint. Our front-end developers might be confused by current naming

Reply:
Fixed